openwisp_monitoring/device/tests/test_models.py

```diff
@@ -306,7 +306,8 @@ def test_init(self):
         self.assertEqual(dd.data, self._sample_data)
 
     def test_retention_policy(self):
-        rp = get_db().get_list_retention_policies()
+        rp = timeseries_db.get_list_retention_policies()
+        # TODO: Test is InfuxDB specific currently
```

Comment:
move all the influxdb specific tests to a new test file in `/openwisp_monitoring/db/backends/influxdb/tests.py`